migration/migrations/course/20180802223149_config_iris_etc.py

```diff
@@ -25,6 +25,8 @@ def up(config, conn, semester, course):
         if not config.has_option('course_details','private_repository'):
             config.set('course_details','private_repository','""')
         # the newest default
+        if not config.has_option('course_details','seating_only_for_instructor'):
+            config.set('course_details','seating_only_for_instructor','false')
```

Comment:
You have to add to this a new migration. Older migrations have already been applied and editing them won't have any affect on existing installations, only new ones.

Reply:
I don't know how to do that... Could you please take a look and help?